src/BaseServiceProvider.php

```diff
@@ -139,10 +139,13 @@ public function publishFiles()
         // publish public Backpack assets
         $this->publishes([__DIR__.'/public' => public_path('vendor/backpack')], 'public');
 
+        // calculate the path from current directory to get the vendor path
+        $vendorPath = __DIR__.'/../../..';
```

Comment:
`$vendorPath = dirname(__DIR__, 3);`

(ok, it doesn't change anything, it's just more elegant :smile:)

Reply:
Agreed! fixed!